src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs

```diff
@@ -15,15 +15,12 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Microsoft.Health.SqlServer.Configs;
-using Microsoft.Health.SqlServer.Features.Schema;
-using Microsoft.Health.SqlServer.Features.Schema.Manager;
 using Microsoft.Health.SqlServer.Features.Schema.Manager.Exceptions;
 using Microsoft.Health.SqlServer.Features.Schema.Manager.Model;
 using Microsoft.SqlServer.Management.Common;
 using Polly;
-using SchemaManager.Core.Model;
 
-namespace SchemaManager.Core;
+namespace Microsoft.Health.SqlServer.Features.Schema.Manager;
 
 public class SqlSchemaManager : ISchemaManager
```

Comment:
Thought it made sense to put it near all the classes and interfaces SqlSchemaManager depends on (BaseSchemaRunner, SchemaClient, SchemaManagerDataStore). 

If you think it would make more sense in the SchemaManager project instead, I'll move them.

Reply:
I think that's reasonable